devices/pine64-pinephone/overlay/default.nix

```diff
@@ -0,0 +1,5 @@
+final: super: {
+  pine64-pinephone = {
+    qfirehose = final.callPackage ./qfirehose {};
+  };
```

Comment:
I'm not sure adding a new top-level attribute for each device is the best solution here. Should it be under `mobile-nixos` instead?